packages/teams-js/test/internal/communication.spec.ts

```diff
@@ -656,6 +656,173 @@ describe('Testing communication', () => {
       }
     });
   });
+  describe('requestPortFromParent', () => {
+    let utils: Utils = new Utils();
+    const actionName = 'test';
+    beforeEach(() => {
+      class MockMessagePort {}
+      global.MessagePort = MockMessagePort as unknown as typeof MessagePort;
+      utils = new Utils();
+      communication.uninitializeCommunication();
```

Comment:
nit: these calls should be able to be replaced with a call to `app._uninitialize`

Reply:
Hmm, I tried this, and 3 of the tests started failing - this is the pattern used by the existing sendMessageToParentAsync tests (where I copied it from :)). 

<img width="447" alt="image" src="https://github.com/OfficeDev/microsoft-teams-library-js/assets/56459261/0c5bac80-2d2d-45c8-b2b8-dfa4dd5dfe3a">

I looked at app._uninit, which does look like it calls to unitializeCommunication, so I don't really know what is happening here...

Reply:
Same as above, I looked into this and it's very weird and not your fault. Okay to leave this as-is in your PR. I'll open a bug to investigate on our end.